src/components/Button.jsx

```diff
@@ -0,0 +1,34 @@
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
```

Comment:
We can just change the material UI component name
```suggestion
import Button as CoreButton from '@mui/material/Button';
```